app-article/src/main/java/top/ruilink/realworld/article/infra/mapper/TestMapper.java

```diff
@@ -0,0 +1,21 @@
+package top.ruilink.realworld.article.infra.mapper;
+
+import org.apache.ibatis.annotations.*;
+import top.ruilink.realworld.article.domain.Test;
+
+@Mapper
+public interface TestMapper {
+
+    @Insert("INSERT INTO tests (created_at, updated_at) VALUES (#{test.createdAt}, #{test.updatedAt})")
```

Comment:
The SQL statements in the TestMapper interface are vulnerable to SQL injection. Although MyBatis provides some protection, it's best practice to ensure that all inputs are properly validated and sanitized before being used in queries.